pkg/scheduler/framework/plugins/podtopologyspread/filtering_test.go

```diff
@@ -1651,7 +1652,7 @@ func TestPreFilterDisabled(t *testing.T) {
 	p := &PodTopologySpread{}
 	cycleState := framework.NewCycleState()
 	gotStatus := p.Filter(context.Background(), cycleState, pod, nodeInfo)
-	wantStatus := framework.NewStatus(framework.Error, `error reading "PreFilterPodTopologySpread" from cycleState: not found`)
+	wantStatus := framework.AsStatus(fmt.Errorf(`error reading "PreFilterPodTopologySpread" from cycleState: not found`))
```

Comment:
remove the word "error" from the message. Same for the rest of the PR.

Reply:
Done